neighbor_finder_test.go

```diff
@@ -45,8 +46,13 @@ func TestBasicExchange(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	nf1 := NewNeighborData(pk1)
-	nf2 := NewNeighborData(pk2)
+	link_local_addr, err := net.ResolveIPAddr("ip6", "fe80::%0")
```

Comment:
It's just a test zone since our neighbor finder tests don't use networking devices at the moment